### ISSUE TITLE
<img> should have alt attributes

### ISSUE THREAD

**Issue by h3xar0n** [jaszan/prj-rev-bwfs-dasmoto#3]
Any img that is not window dressing should have an alt tag for the visually impaired. They use screen readers, and you can convey the main point of the image by using this attribute. For example, if you have a picture of a cup filled with brushes, brushcup.jpg, you could write it like so:

`<img src="./images/brushcup.jpg" alt="A cup filled with brushes">`

Think of the ones you could provide for the following:

https://github.com/jaszan/prj-rev-bwfs-dasmoto/blob/c20571ddbd98ba9f1329d25905b785881a06e958/Dasmoto/index.html#L14

https://github.com/jaszan/prj-rev-bwfs-dasmoto/blob/c20571ddbd98ba9f1329d25905b785881a06e958/Dasmoto/index.html#L23

https://github.com/jaszan/prj-rev-bwfs-dasmoto/blob/c20571ddbd98ba9f1329d25905b785881a06e958/Dasmoto/index.html#L32
